ci-operator/step-registry/cucushift/upgrade/toimage/cucushift-upgrade-toimage-commands.sh

```diff
@@ -792,7 +793,7 @@ do
     if ! check_signed; then
         echo "You're updating to an unsigned images, you must override the verification using --force flag"
         FORCE_UPDATE="true"
-        if check_ota_case_enabled "OCP-30832" "OCP-27986" "OCP-24358" "OCP-69968"; then
+        if check_ota_case_enabled "OCP-30832" "OCP-27986" "OCP-24358" "OCP-69968" "true"; then
```

Comment:
Hmm, I think `ENABLE_OTA_TEST: "true"` conflicts with `check_ota_case_enabled`, `ENABLE_OTA_TEST: "true"` will blindly execute all the OCP-xxx cases, while not every case need to run with a signed image.

Reply:
Sounds your case is not suitable with `ENABLE_OTA_TEST: "true"`, but `ENABLE_OTA_TEST: "OCP-XXXX"`

Reply:
oh ok, Thanks @jianlinliu